src/main/java/Converter.java

```diff
@@ -1,28 +1,86 @@
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.Response;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 
 public class Converter {
```

Comment:
no converter anymore.
seperate the main class to be only executor for the system.
change this class to be called "serverData" class and will implement "getBranchesFromDb"
add a class named "Repository" with some "getBranches" method that will call "getBranchesFromDb" and returns the data.

What's the idea?
Let's say im the frontend developer - I want to get all Branches objects.
As a frontend developer I don't care if you give it to me from the DataBase, from some local memory cache, or whatever. I just want to get the data.
So ill ask from the Repository -> "getBranches". you will decide how to implement it - in our case ofcourse it would be - go to the server and get the data from shauli.

Tell me if its clear